apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -49,41 +51,27 @@ const Select = <TFieldValues extends FieldValues, TOptionValue>({
 		styles["label-text"],
 		isLabelHidden && "visually-hidden",
 	);
+	const option = [field.value]
+		.flat()
+		.map((value) => options.find((option) => option.value === value))
+		.filter(Boolean);
 
```

Comment:
Why do we need to wrap filed.value in array and then use flat ? Can't we use TOptionValue[]?

Reply:
Btw, in JS we have the flatMap method(just for your information)

Reply:
changed, how about now?